CRBM-Viz/src/app/app.module.ts

```diff
@@ -59,6 +64,12 @@ import { ProfileComponent } from './Components/profile/profile.component';
     LinksComponent,
```

Comment:
These have already been added to master I believe so a merge may be needed to work out the changes